### DISCUSSION TITLE
Smart Contract marketplace identifier tag

### DISCUSSION THREAD

**Discussion by Hofstud** [hashlips-lab/nft-erc721-collection#344] [Q&A]
Hey everyone,

Can someone please confirm that line 31 in the smart contract .sol file from the Masterclass collection is the name of the contract file itself that was renamed?

marketplaceIdentifier: "my-nft-token" ???

It may seem obvious but I am getting issues with Opensea and Daniel skips over this line in the video :)

Is it the token symbol, token name, or contract name or name of the .sol file itself??
I am about to deploy on the MainNet Ethereum so want to get this right :)

Thanks so much for all of your help and love the Hashlips community
Leonie

**Comment by liarco** [comment]
Hi @Hofstud,
Here are lines 30 and 31 of the contract:
https://github.com/hashlips-lab/nft-erc721-collection/blob/e2ae0ccea05c3d4a84445c4660538f722c7486a1/smart-contract/contracts/YourNftToken.sol#L30-L31

These lines can be configured to taste in the configuration file and those are human-readable details about your collection. The contract name is a different thing and requires a specific format, while both the name and symbol can be whatever you wish. [Here](https://forum.openzeppelin.com/t/in-erc20-what-are-differences-between-name-and-symbol/1803) you can find further information about these two values.

The values can be configured here:
https://github.com/hashlips-lab/nft-erc721-collection/blob/e2ae0ccea05c3d4a84445c4660538f722c7486a1/smart-contract/config/CollectionConfig.ts#L13-L14

The marketplace identifier is the "slug" on OpenSea. Usually you get this once your collection is created by minting the first token. You can leave it as default (or set it to a random value) and then replace it once the collection gets published on OpenSea.

Please remember: when you change it, you have to rebuild the dApp and publish it again.
